dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -402,6 +379,10 @@ private final class BenchThread implements Callable<Void> {
     private final byte[] mBuffer;
     private final WorkerBenchTaskResult mResult;
     private final boolean mIsRandomRead;
+    private final Random mRandom;
+    private final int mRandomMax;
+    private final int mRandomMin;
+    private final int mFileSize;
```

Comment:
using int for file sizes limits the file to be smaller that 2.1GB. Can you either add a check to the cli parser to disallow input of files larger than 2.1GB, or change it to `long`?

Reply:
consequently, `offset` should be check in the same way as well.

Reply:
all changed to long, and StressWorkerBench will make sure the MaxReadLength is within 2.1G when validating the parameters. since the InputStream.read() method only support read length in integer.